reana_workflow_engine_yadage/externalbackend.py

```diff
@@ -22,20 +22,12 @@
 log = logging.getLogger(LOGGING_MODULE)
 
 
-def make_oneliner(job):
-    """Convert a command into oneliner."""
-    wrapped_cmd = "sh -c {}  ".format(pipes.quote(job["command"]))
-    return wrapped_cmd
-
-
 def make_script(job):
```

Comment:
Background why this is not removed yet https://github.com/reanahub/reana-job-controller/pull/305#pullrequestreview-613367088.